urc_platform/src/main.cpp

```diff
@@ -0,0 +1,13 @@
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp/time.hpp>
+#include <rclcpp_components/register_node_macro.hpp>
+
+int main(int argc, char ** argv)
```

Comment:
Why are you using single-threaded executors? Do you really need this for the node to function